GoogleForCreators/web-stories-wp
# Replace `@wordpress/date` with `php-date-formatter`

## Summary

We currently use the `@wordpress/date` package for date formatting.

This causes a couple of issues:

* We load it as an **external** (meaning it is not part of the actual bundle) because it is GPL licensed...
* ...which means it itself loads `moment` as an external since it's a dependency.
* An additional hurdle for decoupling the app from WordPress (#2918)

## Relevant Technical Choices

Replaces `@wordpress/date` with `php-date-formatter`

## To-do

* [ ] Don't load `moment` itself as an external, and make it part of the bundle instead.  
  Need to ensure we still set up `moment` locale configuration as currently.

## User-facing changes

None.

## Testing Instructions

Change date formatting in settings, verify that dates are displayed as expected in dashboard (my stories view) & editor (media edit dropdown).

---



See #2199
